features/admin/redirecting_on_login_after_being_logout.feature

```diff
@@ -10,5 +10,5 @@ Feature: Redirecting on login page
 
     @ui
     Scenario: Redirecting on login page after being logout
-        When I try to open administration dashboard
+        When I browse administration dashboard statistics
```

Comment:
```suggestion
        When I try to browse administration dashboard statistics
```

Just a cosmetic change. It can be ignored if no more significant changes are suggested.